addons/account_sepa/account_sepa_credit_transfer.py

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8 -*-
+
+import re
+import time
+import random
+import base64
+
+from unidecode import unidecode
+
+from openerp import models, fields, api, _
+from openerp.tools import float_round
+from openerp.exceptions import UserError, ValidationError
+
+''' TODO
+    - Improve res.partner.bank
+    - Check base_iban is reliable (eg. get_bban_from_iban contains a TODO)
```

Comment:
But maybe we could support the 'strict SEPA' only. In this case, it would only be possible to make euro payments inside the SEPA area using valid IBAN and BIC codes, but the specs and code would be much less ambiguous.